account_customer_wallet/models/res_company.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 Coop IT Easy SCRL fs
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class Company(models.Model):
+    _inherit = "res.company"
+
+    customer_wallet = fields.Boolean(
+        string="Customer Wallet Enabled",
+        default=False,
+    )
+    customer_wallet_account_id = fields.Many2one(
+        comodel_name="account.account",
+        string="Customer Wallet Account",
+        # TODO: Filter domain on type?
```

Comment:
I don't think so. i think this will depends on localization or point of views of accountant. So let free is the better / simplier option.